appveyor.yml

```diff
@@ -65,8 +66,6 @@ before_build:
   cmd: >-
     py -m pip install nihtest
 
-    vcpkg install
```

Comment:
This is not required in manifest mode, CMake will take care of this install automatically